homeassistant/components/lifx/binary_sensor.py

```diff
@@ -50,15 +50,8 @@ def __init__(
         super().__init__(coordinator)
         self.entity_description = description
         self._attr_unique_id = f"{coordinator.serial_number}_{description.key}"
-        self._async_update_attrs()
 
-    @callback
-    def _handle_coordinator_update(self) -> None:
-        """Handle updated data from the coordinator."""
-        self._async_update_attrs()
-        super()._handle_coordinator_update()
-
-    @callback
-    def _async_update_attrs(self) -> None:
-        """Handle coordinator updates."""
-        self._attr_is_on = self.coordinator.async_get_hev_cycle_state()
+    @property
+    def is_on(self) -> bool | None:
+        """Returns true if a HEV cycle is currently running."""
+        return self.coordinator.async_get_hev_cycle_state()
```

Comment:
Let's leave the platforms as-is in this PR so we can keep this small and maybe backport it later

Reply:
I was afraid you'd say that. Will do.